pkgs/development/ocaml-modules/uuidm/default.nix

```diff
@@ -18,7 +18,8 @@ stdenv.mkDerivation rec {
     description = "An OCaml module implementing 128 bits universally unique identifiers version 3, 5 (name based with MD5, SHA-1 hashing) and 4 (random based) according to RFC 4122";
     homepage = "https://erratique.ch/software/uuidm";
     license = licenses.bsd3;
-    platforms = ocaml.meta.platforms or [];
     maintainers = [ maintainers.maurer ];
+    mainProgram = "uuidtrip";
+    platforms = ocaml.meta.platforms or [];
```

Comment:
```suggestion
    platforms = ocaml.meta.platforms;
```

Reply:
I noticed that there were many more cases of this in `ocamlPackages` so I've gone ahead and fixed them all by changing them to `inherit (ocaml.meta) platforms;` in a new commit.